x11-misc/libinput-gestures/libinput-gestures-9999.ebuild

```diff
@@ -26,7 +30,17 @@ DEPEND=">=dev-libs/libinput-1.8.0
 	dev-util/desktop-file-utils
 	test? ( dev-python/flake8[${PYTHON_USEDEP}] )"
 
+src_prepare() {
+	default
+
+	# Fix docdir installation path
+	sed '/^DOCDIR/s@$NAME@${PF}@' -i libinput-gestures-setup || die
+}
+
 pkg_postinst() {
+	gnome2_icon_cache_update
+	xdg_desktop_database_update
```

Comment:
There is no "MimeType=" in libinput-gestures.desktop.
Looks like xdg_desktop_database_update shouldn't be here.
https://wiki.gentoo.org/wiki/Notes_on_ebuilds_with_GUI

Reply:
Fair enough. I gonna adjust the commit accordingly.